webpack/envs/clientCommonConfig.js

```diff
@@ -1,138 +1,29 @@
 // @ts-check
 
-const path = require("path")
-const webpack = require("webpack")
-const LoadablePlugin = require("@loadable/webpack-plugin")
-const { RetryChunkLoadPlugin } = require("webpack-retry-chunk-load-plugin")
-const { basePath, env } = require("../utils/env")
+import LoadablePlugin from "@loadable/webpack-plugin"
+import path from "path"
+import { basePath, env } from "../utils/env"
+import {
+  clientExternals,
+  standardDevtool,
+  standardMode,
+  standardResolve,
+  standardStats,
+} from "./commonEnv"
```

Comment:
Can we rename this `webpackEnv`? With `env` being used extensively throughout, `commonEnv` / `env` is confusing.

Reply:
I'll think on it and see if I can come up with a more suitable name. It's unlikely that naming will be the main point of confusion for those working with Webpack.

Reply:
If there's `env` and `commonEnv` side by side, that's def confusing. But if `commonEnv` actually refers to `webpackEnv` then problem solved.